arm/Microsoft.DataFactory/factories/deploy.bicep

```diff
@@ -125,26 +134,21 @@ resource dataFactory 'Microsoft.DataFactory/factories@2018-06-01' = {
   }
 }
 
-resource dataFactory_managedVirtualNetwork 'Microsoft.DataFactory/factories/managedVirtualNetworks@2018-06-01' = {
-  parent: dataFactory
-  name: 'default'
-  properties: {}
+module dataFactory_managedVirtualNetwork 'managedVirtualNetwork/deploy.bicep' = {
```

Comment:
Is this child mandatory? If not I'd suggest to add a condition to the resource such as:

```bicep
module dataFactory_managedVirtualNetwork 'managedVirtualNetwork/deploy.bicep' = if (!empty(managedVirtualNetworkName)) {
```

Reply:
On of course make the parameter non-mandatory in this file too. However, do this only if the resource is actually not required

Reply:
Not mandatory, this has been fixed